tests/helpers.go

```diff
@@ -170,6 +171,7 @@ func executeTest(t *testing.T, testFile string) {
 				}
 				t.Fatalf("unknown test issue: %s, output: %s", err.Error(), string(out))
 			}
+			require.Fail(t, err.Error())
```

Comment:
The current check:

```go
var exitError *exec.ExitError
if errors.As(err, &exitError)
```

is not enough to properly fail the testing workflow.

Because the CI workflow did not run `npm install`, the error was of type `*fs.PathError` and the error messages was:
```bash
fork/exec ./web3js/node_modules/.bin/mocha: no such file or directory
```